hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -1062,6 +1062,9 @@ public void run() {
           }
         } catch (IOException | InterruptedException ex) {
           LOG.error("Exception while validating write: " + ex.getMessage());
+          if(ex instanceof InterruptedException) {
```

Comment:
The `instanceof` check triggers another warning:

> Exception types should not be tested using "instanceof" in catch blocks
>
> Code smell    Major    squid:S1193
>
> Multiple catch blocks of the appropriate type should be used instead of catching a general exception, and then testing on the type.

Reply:
@adoroszlai Addressed review comments in latest commit. 
Did you run the sonar check locally to discover this?
I want to understand so in future I can take care of these before I open a PR :)

Reply:
Thanks for updating the patch.

> Did you run the sonar check locally to discover this?

Yes, using [SonarLint](https://www.sonarlint.org/) plugin.  But this one I also remembered from another bug.

> I want to understand so in future I can take care of these before I open a PR :)

Thanks for that. :)